framework/src/utils/FillBetweenPointVectorsTools.C

```diff
@@ -290,7 +290,7 @@ elementsCreationFromNodesVectors(ReplicatedMesh & mesh,
       // Define the two possible options and chose the one with shorter distance
       Real dis1 = (*nodes[i + 1][nodes_up_it] - *nodes[i][nodes_down_it + 1]).norm();
       Real dis2 = (*nodes[i + 1][nodes_up_it + 1] - *nodes[i][nodes_down_it]).norm();
-      if (dis1 > dis2)
+      if (MooseUtils::absoluteFuzzyGreaterThan(dis1, dis2))
```

Comment:
The original expression sometimes causes different meshes when using different numbers of cores due to the precision issue.

Reply:
nice thanks for the bug fix!